_component_design/assisted-search.md

```diff
@@ -44,40 +44,83 @@ When the user clicks the search box, show them a list of keywords that they can
 
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### Layout specifications
+### Active state
 
-Separate groups of keywords using a horizontal divider.
+When the search box is clicked and the cursor is active, a designer may choose to prompt the user with recent or suggested searches depending on their context. Recent Searches is just one example of content that can populate an Assistive Search Box. 
 
-Append the keyword with colon to the search input when the user clicks on a keyword in the menu.
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Assisted search specifications"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/search-and-filters/search-assisted/assisted-search-active.png){: width="633px"}
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Post-entry state
+
+Follow the Search Box conventions for entered text within the text box. As text is entered in the search box, that text is represented in the Assistive Search Box in the first line: “Search for ‘item’”. The “X” to remove the search item appears as soon as any text is entered.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Assisted search post-entry"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/search-and-filters/search-assisted/assisted-search-post-entry.png){: width="633px"}
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
+### Loading
+
+Use a [loading spinner]({{site.baseurl}}/components/loading-indicators.html) if results take need to be processed prior to display. As a default, use a height of 248px.
+
+User should still have instructions for how to search without autocompleted results (Enter).
```

Comment:
Do you mean the Enter key?

Reply:
I don't know what this means, I'll ask the writer.

Reply:
Chatted with @racksmangonzalez, and looks like this sentence referenced an older version of the design. It will be omitted.